style.css

```diff
@@ -0,0 +1,102 @@
+.sort {
+    width: 8px;
+}
+
+.heart {
+    width: 25px;
+}
+
+.user {
+    width: 22px;
+    opacity: 0.7;
+}
+
+/* .heart:hover {
+    height: 30px;
+    width: 25px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+  } */
+
+header {
+    max-width: 1200px;
+    height: 80px;
+    margin-left: 31.667px;
+    margin-right: 31.667px;
+}  
+
+nav{
+    padding-top: 20px;
+    padding-bottom: 20px;
+    color: #053361;
+}
+
+header a{
+    color: #000000;
+    text-decoration: none;
+    font-size: 16px;
+}
+
+header ul{
+    margin: 0;
+    padding: 0;
+}
+
+.right{
+    float: right;
+
+}
+
+.right img {
+    margin: 0 0 0 5px;
+}
+
+.right a {
+    margin: 0 0 0 9px;
+}
+
+.left{
+    float: left;
+}
+
+.left a {
+    margin: 0 0 0 9px;
+}
+
+.header {
+    font-size:35px;
+    padding-top: 5px;
+    font-weight: bold;
+    color: #053361;
+}
+
+.search-bar{
```

Comment:
@Afrocentriccynic When you take out the search bar, remember to remove the corresponding css.